### ISSUE TITLE
Update to Latest Extreme Reactors

### ISSUE THREAD

**Issue by SSyl** [NillerMedDild/Enigmatica2Expert#1646]
I'm not sure if there's a reason why you're staying on the 0.4.5.50. I do know the mod was updated to use proper block names rather than meta data, so I went ahead and updated the mod to the latest version 0.4.5.67 and fixed all scripts to use the proper block names.

I've been running it for about a week now without any conflicts or issues arising.

Along with this, you have to update ZeroCore to its latest version and you can optionally update Just Enough Reactors as well.

Related to this pull request: #1645

**Comment by NillerMedDild** [comment]
Erhm... this would remove existing blocks that get new registry names, unless there's some error handling for that?

**Comment by MorganSimic** [comment]
From the changelog is seems like there are DataFixers in place. That should fix most of the stuff but probably has one big flaw. It most certainly will not affect stuff in AE systems or similar only vanilla-ish inventories and in-world. So reactors woild probably be safe but tons of the ludicrite you may have on disks are not. At least that's what i gathered from the changelog.

**Comment by NillerMedDild** [comment]
I cannot accept the PR then I'm afraid, but I appreciate the effort @SSyl :)

**Comment by SSyl** [comment]
No worries! I'm using it on my own fork of E2E with a friend and I had updated the mod + scripts before we got to the Extreme Reactors stage, so that explains why we didn't get any sort of block/item loss.

I do appreciate that you looked into it! I hope I'm not bombarding you with requests that waste your time since I know you have a boatload of modpacks to manage plus your own personal/professional life, so I really do appreciate the time you take looking into things.
